wemod_launcher/utils/consts.py

```diff
@@ -0,0 +1,17 @@
+import os
+from .configuration import Configuration
+from pathlib import Path
+
+
+class Consts:
+    CFG = Configuration()
+    SCRIPT_RUNTIME_DIR: Path = Path(
+        os.path.dirname(os.path.realpath(__file__))
+    ).expanduser()
```

Comment:
i don't think .expanduser() is needed here os.path.realpath should already give a absolute path

Reply:
Agreed, fixed.